commands/system/toggle-do-not-disturb.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+# Dependency: This script requires `calm-notifications` cli installed: https://github.com/vitorgalvao/tiny-scripts/blob/master/calm-notifications
+# Install via homebrew: `brew install vitorgalvao/tiny-scripts/calm-notifications`
+
+# Required parameters:
+# @raycast.schemaVersion 1
+# @raycast.title Toggle Do Not Disturb
+# @raycast.mode compact
+#
+# Optional parameters:
+# @raycast.icon 🔕
+# @raycast.packageName System
+#
+# @Documentation:
+# @raycast.description Toggle Do Not Disturb
+# @raycast.author Antonio Dal Sie
+# @raycast.authorURL https://github.com/exodusanto
+
+if [ "on" = $(calm-notifications status) ]
+then
+  calm-notifications off
+  # Wait 2s for status bar apply
+  sleep 2
+  echo "Do Not Disturb off"
+else
+  calm-notifications on
+  echo "Do Not Disturb on"
+fi
```

Comment:
Seems like `calm-notifications` also supports the `toggle` command, wouldn't the following be simpler?

```suggestion
calm-notifications toggle

echo "Do Not Disturb $(calm-notifications status)"
```

Reply:
This even makes me wonder whether we should accept `on/off/toggle/status` as parameter and pass it along 🤔

Reply:
Original implementation was `calm-notifications toggle`. In the `off` case the DND Icon on status bar change opacity and disappear after some seconds.
That code is to wait some seconds in that case to prevent double toggle request.

But maybe is useless in most cases 😅

Reply:
Ah, that makes sense I guess! I'm fine with leaving it at is for now, and see if people like it this way. We can always accept future PRs if need be :-)

Reply:
The parameter i see that `default` attribute is not implemented yet, and in my idea the script is a shortcut to toggle. This can be an enhancement with that option in the future 🤗

Reply:
I was thinking that if you make the parameter optional you can default to toggling in Bash